rd_server.py

```diff
@@ -103,7 +103,11 @@ def run(self):
         run.rundir = config['runs'] + '/' + run_id
         run.log = log_file
         run.set = info['task']
-        run.bindir = run.rundir + '/x86_64/'
+        if 'arch' in info:
+            run.arch = info['arch']
+        else:
+            run.arch = 'x86_64'
```

Comment:
you could also just do:
run.arch = info.get('arch', 'x86_64')

Reply:
fixing in next patch